src/app/api/tasks/tasks.service.ts

```diff
@@ -665,4 +674,15 @@ export class TasksService extends BaseService {
     })
     return !!nextTask
   }
+
+  canCreateSubTask(parentPath: string): boolean {
+    //regex to match ltree path format, underscore and no dashes.
+    const uuidRegex = /\b[0-9a-fA-F]{8}_[0-9a-fA-F]{4}_[0-9a-fA-F]{4}_[0-9a-fA-F]{4}_[0-9a-fA-F]{12}\b/g
+
+    const uuids = parentPath.match(uuidRegex)
+
```

Comment:
I would just be easier to do 
```suggestion
    const uuidLength = parentPath.split('.').length
```